src/components/ConditionalWrapper/index.jsx

```diff
@@ -0,0 +1,4 @@
+const ConditionalWrapper = ({ condition, wrapper, children }) =>
+  condition ? wrapper(children) : children;
+
+export default ConditionalWrapper;
```

Comment:
Is it being used anywhere? Any dead code / packages / resources (images, etc.) should be removed.

Reply:
Conditional wrapper is used to remove the column resize when on mobile. moved it to the App.jsx code since its small